authappliance/menu.py

```diff
@@ -1719,6 +1783,9 @@ def main_menu(self):
                         "Backup and Restore", "",
                         "Backup or Restore of privacyIDEA "
                         "configuration and database."))
+        choices.append((self.updatesDialog.menu,
+                        "Automatic Updates", "",
```

Comment:
Lets change this to "Updates" only.
This way each menu entry has a unique first character.